mslib/msui/msui_web_browser.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+"""
+    mslib.msui.msui_web_browser.py
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    QTWebBrowser for the SSO process.
```

Comment:
the formatting is quite condensed but not that important

after a module import you do read it by help()

```
from mslib.msui import msui_web_browser
help(msui_web_browser)
```